src/Npgsql.NetTopologySuite/NpgsqlNetTopologySuiteExtensions.cs

```diff
@@ -38,18 +38,32 @@ namespace Npgsql
     /// </summary>
     public static class NpgsqlNetTopologySuiteExtensions
     {
+        static readonly Type[] s_clrTypes = new[]
```

Comment:
nit: we don't use this naming convention in the Npgsql code base, static readonly variables are generally named `ClrTypes`. Do you mind fixing this please (and also in the GeoJSON plugin)?

Reply:
Done.